dist/s_tags

```diff
@@ -1,41 +1,37 @@
 #! /bin/sh
 
-# Build tags file.
-#
-t=__a.c
-trap 'rm -f $t; exit 0' 0 1 2 3 13 15
+# Build tags files.
+trap 'rm -f tags; exit 0' 0 1 2 3 13 15
 
```

Comment:
@keithbostic, If this script is run while current directory is build_posix or src/include, the resulting tags file will be removed.  Probably unlikely anyone would do that.  Maybe use `ctags -f tmptags` when test the ctags options?